gibbon-compiler/gibbon.cabal

```diff
@@ -127,25 +127,22 @@ test-suite test-gibbon
   hs-source-dirs:      tests
   main-is:             Main.hs
 
-  build-depends:       base
-                     , gibbon
-                     , containers
-                     , filepath
-                     , directory
-                     , process
-                     , mtl
-                     , srcloc
-                     , tasty
-                     , tasty-hunit
-                     , tasty-th
-
-  other-modules:       RouteEnds
-                       InferEffects
-                       Unariser
-                       Compiler
-                       L2.Typecheck
-                       L1.Typecheck
-                       L3.Typecheck
+  build-depends:       base,
+                       gibbon, containers,
+                       filepath, directory, process,
+                       mtl,
+                       srcloc,
+                       mtl, transformers,
```

Comment:
We could drop `mtl` here, since it already appears on L133.

Reply:
Good catch. That was probably introduced during a merge.